addons/sale/i18n/sale.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 17.0\n"
+"Project-Id-Version: Odoo Server 17.0+e\n"
```

Comment:
This is also wrong, means you exported the pot with enterprise installed :eyes:
You can remove the changes in the file headers, I was never told they were necessary, and they'll bring conflicts in every forward-port :crying_cat_face: